client/src/utils/nominatimApi.js

```diff
@@ -0,0 +1,24 @@
+const axios = require("axios");
+
+const instance = axios.create({
+  baseURL: "https://nominatim.openstreetmap.org",
+});
+
+export const getBoundsOfState = (place) => {
+  const requestExtension = `search?q=${place}&format=jsonv2&polygon_geojson=1`;
```

Comment:
I think there is a state parameter that you can use instead to query exact results instead of a large series. We will use the `q` param for grabbing more granular locations so you can actually just rename this function and then create one for specifically grabbing the state.